src/feedspora/feedspora_runner.py

```diff
@@ -345,14 +315,21 @@ def shorten_url(self, the_url):
                                        self._url_shortener)
             # pylint: disable=broad-except
             except Exception as exception:
```

Comment:
Here, you catch a generic exception, then branch the flow with a `if` statement in the `except` block. Python supports multiple `except` statements (see [here](https://www.tutorialspoint.com/python3/python_exceptions.htm) for an example). This is both more robust and readable. The corresponding code might be (not tested):

```
try:
    ....
except AssertionError:
    logging.error("URL shortenener %s not implemented", self._url_shortener)
    ....
except AttributeError:
    logging.error("URL shortenener %s raised an error", self._url_shortener)
    ....
except Exception:
    # We should never be here, probably we don't want such an unspecific except clause!

```

Reply:
I implemented it the way I did because ANY exception raised needs to be treated the same way (revert to the original URL).  It seems problematic, redundant, and risky from a code maintenance standpoint to have separate exception branches with the exact same action occurring, with only their logging messages being different.
However, if you insist I'll make that change.

Reply:
Good point. There are probably more elegant solution but the one you suggest seems to be good enough :-)